### PR TITLE
LRQA-68967 FormsTextField.testcase tests refactor

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by leticiachiu** [comment]
ci:test:sf

**Comment by leticiachiu** [comment]
ci:test:sf

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by leticiachiu** [comment]
@natocesarrego fixed

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase [hunk 1]
```diff
@@ -436,6 +847,78 @@ definition {
...
+		FormsAdminNavigator.gotoViewEntries();
+
+		FormsAdmin.viewFormTableEntry(
+			fieldLabel = "Text Field",
+			fieldValue = "Text Field Value");
+	}
+
 	@description = "Verify that a Text Field can require Confirmation"
 	@priority = "5"
 	test CanRequireConfirmation {
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase:924]: @leticiachiu, can you please rename once this test has same name of the test created by you?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase [hunk 2]
```diff
@@ -436,9 +847,81 @@ definition {
 			successPageTitle = "Thank you.");
 	}
 
-	@description = "Verify that a Text Field can require Confirmation"
+	@description = "LPS-125860 - Verify that Confirmation field property is working as expected"
 	@priority = "5"
 	test CanRequireConfirmation {
+		property portal.acceptance = "true";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase:853]: @leticiachiu, should it be in the acceptance suite? Can you please confirm with Diogo?

Comment (leticiachiu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase:853]: I just moved Forms#SubmitFormWithRequiredConfirmationTextField to FormsTextField.testcase, so I kept the property.. is that right, @diogocordeiro?

Comment (natocesarrego) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase:853]: Makes sense, @leticiachiu! :slightly_smiling_face:

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**